Dockerfile

```diff
@@ -13,6 +13,13 @@ WORKDIR /go/src/github.com/minio/
 
 COPY dockerscripts/docker-entrypoint.sh dockerscripts/healthcheck.sh /usr/bin/
 
+# add our user and group first with fixed uid and gid, this ensures
+# multiple updates to the minio image dont have permission issues.
+RUN addgroup -g 190 -S minio && adduser -u 190 -S -G minio minio
+
+# grab su-exec for easy step-down from root
+RUN apk add --no-cache 'su-exec>=0.2'
```

Comment:
you can add this to the below command as well, we can avoid adding another docker layer here. Also this needs to be changed across Dockerfile.release, Dockerfile.dev as well.  Installation should be all together.